CHANGELOG.md

```diff
@@ -1,5 +1,21 @@
 # Changelog
 
+## 0.21.0
+
+### Breaking Changes
+
+- `initTestHelpers()` setup has to be removed with Jest/Jasmine-like test runners
```

Comment:
I'm not quite understanding this change -- we are still exporting `initTestHelpers` and if users are using it there is no harm.

Reply:
No big harm, I just wanted to highlight that the setup routine has changed. I should rephrase it.